lambdaclass/mirra_backend
# [GH-852] Configurator: Migrate traps config

## Motivation

Keep migrating fields from config.json to configurator app

Part of https://github.com/lambdaclass/mirra_backend/issues/852

## Summary of changes

- Added logic to hable items mechanics
- Added fake_item to game seeds
- Added show and edit of mechanics in consumable items to configurator app

## How to test it?

turn the fame item to active in the game configurator and then play a match and use that item, it should spawn a new entity that deals damage after a while if someone steps on it

## Checklist
